jwala-integration-test/src/test/resources/postman/server.xml.tpl

```diff
@@ -91,8 +91,8 @@
 
     <Connector 
       port="${jvm.httpsPort}" 
-	  SSLCertificateFile="${vars['remote.paths.deploy.dir']}/app/data/security/id/${jvm.hostName}.cer"
-      SSLCertificateKeyFile="${vars['remote.paths.deploy.dir']}/app/data/security/id/${jvm.hostName}.key"
+	  SSLCertificateFile="${vars['remote.jwala.data.dir']}/../../app/data/security/id/${jvm.hostName}.cer"
+      SSLCertificateKeyFile="${vars['remote.jwala.data.dir']}/../../app/data/security/id/${jvm.hostName}.key"

```

Comment:
Have to make sure this also has the .tokenize('.')]0].downcase... hopefully the merge takes care of that.